pkg/controller/factory.go

```diff
@@ -157,7 +157,5 @@ func (f *GardenControllerFactory) Run(stopCh <-chan struct{}) {
 	logger.Logger.Infof("============================")
 	tw.Flush()
 	logger.Logger.Infof("============================")
-	logger.Logger.Infof("Bye bye!")
-
-	os.Exit(0)
```

Comment:
Since this has been removed, stopping the controllers w/o leader election will cause a panic.